docs/reference/edgectl-download.md

```diff
@@ -0,0 +1,42 @@
+# Downloads
+
+You can download `edgectl`, the executable for Edge Control on a macOS, Linux, or
```

Comment:
I prefer a more active voice over a more passive voice. What I mean is instead of "You can download..." I prefer a wording of "Edge Control is available on Linux, macOS, and Windows".